Maths/CircularArc.js

```diff
@@ -6,36 +6,36 @@
  * @see https://en.wikipedia.org/wiki/Degree_(angle)
  * @example degreesToRadians(45) = 0.7853981633974483
  */
- function degreesToRadians(degrees) {
-    return angle = degrees * Math.PI / 180
-}
+ function degreesToRadians (degrees) {
+    return degrees * Math.PI / 180
+  }
 
-/**
- * @function circularArcLength
- * @description calculate the length of a circular arc
- * @param {Integer} radius 
- * @param {Integer} degrees 
- * @returns {Integer} radius * angle_in _adians
- * @see https://en.wikipedia.org/wiki/Circular_arc
- * @example circularArcLength(3, 45) = 2.356194490192345
- */
-function circularArcLength(radius, degrees) {
-    return length = radius * degreesToRadians(degrees)
-}
-/**
- * @function circularArcArea
- * @description calculate the area of the sector formed by an arc
- * @param {Integer} radius 
- * @param {Integer} degrees 
- * @returns {Integer} 0.5 * r * r * angle_in_radians
- * @see https://en.wikipedia.org/wiki/Circular_arc
- * @example circularArcArea(3,45) = 3.5342917352885173
- */
-function circularArcArea(radius, degrees) {
-    return area = Math.pow(radius, 2) * degreesToRadians(degrees) / 2
-}
+  /**
+   * @function circularArcLength
+   * @description calculate the length of a circular arc
+   * @param {Integer} radius
+   * @param {Integer} degrees
+   * @returns {Integer} radius * angle_in _adians
+   * @see https://en.wikipedia.org/wiki/Circular_arc
+   * @example circularArcLength(3, 45) = 2.356194490192345
+   */
+  function circularArcLength (radius, degrees) {
```

Comment:
why are you indenting the functions now?

Reply:
I thought it would be better if the code followed the standard style

Reply:
The standard style mandates indenting top level functions? I doubt that.